### ISSUE TITLE
Smart Fox Server 2X Community Edition support

### ISSUE THREAD

**Issue by HeadClot** [armory3d/armory#17]
Hey @luboslenco -

This is a feature request for a more advanced server architecture for more intensive Armory3D games. We are looking at [Smart Fox Server 2X Community Edition](http://www.smartfoxserver.com/). Which has a [HAXE integration](https://github.com/chapatiz/smartfox-haxe-client) currently for the client. It is worth noting that you would not be able to use the community edition commercially. So It would probably be best to just make some bindings for Smart Fox Server X2.

In short - This would be an nice extra feature when Armory3D reaches a more mature state.

**Comment by Justinfront** [comment]
I don't see how this relates to Armory3D, from my understanding you can hook up whatever haxe you want.  Seems Haxe has smart server already I don't think this is Armory specific related as you also look at CastleDB.
https://github.com/chapatiz/smartfox-haxe-client

**Comment by HeadClot** [comment]
Going to close this issue so that It does not clog up the Feature Requests and bugs :)
